packages/gatsby/src/utils/get-server-data.ts

```diff
@@ -0,0 +1,46 @@
+import type { Request } from "express"
+import type { IGatsbyPage } from "../redux/types"
+
+import { match } from "@gatsbyjs/reach-router/lib/utils"
+
+export interface IServerData {
+  headers?: Record<string, string>
+  props?: Record<string, unknown>
+}
+
+interface IModuleWithServerData {
+  getServerData?: (args: {
+    headers: Map<string, unknown>
+    method: string
+    url: string
+    query?: string
```

Comment:
I'm not sure this is correct typing? Shouldn't this be `Record<string,string>`?

Reply:
you're right :)